x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/hosts_dashboard.json

```diff
@@ -0,0 +1,1482 @@
+                          "includeEmptyRows": true,
+                          "dropPartials": false
+                        }
+                      },
+                      "cc8d6fed-7ad0-4aa4-9bdf-46526647dee5X0": {
+                        "label": "Part of Memory Usage",
+                        "dataType": "number",
+                        "operationType": "average",
+                        "sourceField": "system.memory.actual.used.pct",
+                        "isBucketed": false,
+                        "scale": "ratio",
+                        "params": {
+                          "emptyAsNull": false
+                        },
+                        "customLabel": true
+                      },
+                      "cc8d6fed-7ad0-4aa4-9bdf-46526647dee5": {
+                        "label": "Memory Usage",
+                        "customLabel": true,
+                        "operationType": "formula",
+                        "dataType": "number",
+                        "references": [
+                          "cc8d6fed-7ad0-4aa4-9bdf-46526647dee5X0"
+                        ],
+                        "isBucketed": false,
+                        "params": {
+                          "formula": "average(system.memory.actual.used.pct)",
+                          "format": {
+                            "id": "percent",
+                            "params": {
+                              "decimals": 1
+                            }
+                          },
+                          "isFormulaBroken": false
+                        }
+                      }
+                    },
+                    "columnOrder": [
+                      "11238e67-baeb-41f4-aebf-ea90e431723a",
+                      "cc8d6fed-7ad0-4aa4-9bdf-46526647dee5X0",
+                      "cc8d6fed-7ad0-4aa4-9bdf-46526647dee5"
+                    ],
+                    "sampling": 1,
+                    "ignoreGlobalFilters": false,
+                    "incompleteColumns": {}
+                  }
+                },
+                "currentIndexPatternId": "infra_metrics_8bf7e99f-ca01-5a18-b70d-573d73a17bb6"
+              }
+            },
+            "internalReferences": [
+              {
+                "type": "index-pattern",
+                "id": "infra_metrics_8bf7e99f-ca01-5a18-b70d-573d73a17bb6",
+                "name": "indexpattern-datasource-layer-layer_0"
+              },
+              {
+                "type": "index-pattern",
+                "id": "infra_metrics_8bf7e99f-ca01-5a18-b70d-573d73a17bb6",
+                "name": "indexpattern-datasource-layer-layer_0_trendline"
+              }
+            ],
+            "adHocDataViews": {
+              "infra_metrics_8bf7e99f-ca01-5a18-b70d-573d73a17bb6": {
+                "id": "infra_metrics_8bf7e99f-ca01-5a18-b70d-573d73a17bb6",
+                "title": "remote_cluster:metricbeat-*,metricbeat-*,remote_cluster:metrics-*,metrics-*",
```

Comment:
The data view is hard-coded here but this value is from the settings page. Is there a way to override these? Just loop through and replace the titles?